src/EventGrid/EventGrid.Test/ScenarioTests/Common.ps1

```diff
@@ -55,7 +55,7 @@ Get Hybrid Connection ResourceId
 #>
 function Get-HybridConnectionResourceId
 {
-    return "/subscriptions/$subscriptionId/resourceGroups/<ResourceGroupName>/providers/Microsoft.Relay/namespaces/<NameSpace>/hybridConnections/<HybridConnectionName>"
+    return "/subscriptions/5b4b650e-28b9-4790-b3ab-ddbd88d727c4/resourceGroups/DevExpRg/providers/Microsoft.Relay/namespaces/DevExpRelayNamespace/hybridConnections/hydbridconnectiondestination"
```

Comment:
@ahamad-MS this is fine since we're in a separate preview branch, but we should make sure that resource ids aren't hard-coded in order to allow any user to come in and record your tests